test/support/builders/metadata/base.js

```diff
@@ -61,6 +61,12 @@ module.exports = class BaseMetadataBuilder {
     return this
   }
 
+  moveTo (id /*: string */) /*: this */ {
+    this.doc.moveTo = id
+    this.doc._deleted = true
```

Comment:
Aren't we duplicating part of `core/move`'s logic with those two lines?

Reply:
Other solutions are not so nice, better duplicate until we find a better way (still better than duplicating in many tests).